pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -192,6 +193,20 @@ type FluentdSpec struct {
 	Tolerations  []v1.Toleration   `json:"tolerations,omitempty"`
 }
 
+type LogMetricExporterSpec struct {
+	// The resource requirements for Fluentd
+	//
+	// +nullable
+	// +optional
+	Resources *v1.ResourceRequirements `json:"resources"`
+
+	// Define which Nodes the Pods are scheduled on.
+	//
+	// +nullable
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
Remove me.  this container will be deployed with fluentd and will be bound to what is defined there

Reply:
resolved